src/main/java/wolox/training/models/User.java

```diff
@@ -55,11 +64,16 @@
   @Setter
   private List<Book> books;
 
-  public User(String username, String name, LocalDate birthDate, List<Book> books) {
+  @Column(nullable = false)
+  @Getter
+  private String password;
```

Comment:
Add `@JsonProperty(access = Access.WRITE_ONLY)` for avoid read when serialization is doing